go.mod

```diff
@@ -18,15 +20,15 @@ require (
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/crypto v0.18.0
 	golang.org/x/sync v0.6.0
-	helm.sh/helm/v3 v3.13.3
-	k8s.io/api v0.28.5
-	k8s.io/apiextensions-apiserver v0.28.5
-	k8s.io/apimachinery v0.28.5
-	k8s.io/cli-runtime v0.28.5
-	k8s.io/client-go v0.28.5
-	k8s.io/component-base v0.28.5
+	helm.sh/helm/v3 v3.14.1
```

Comment:
We also need helm version in helm-operator-plugins repo bumped.

Reply:
There is https://github.com/operator-framework/helm-operator-plugins/pull/301 which still needs a bit of work. I'll take over from Kevin.